src/pages/dataTableUtilities/reducer/reducerMethods.js

```diff
@@ -281,10 +282,16 @@ export const addMasterListItems = state => {
  * Action: { type: 'addItem', item, addedItemType }
  */
 export const addItem = (state, action) => {
-  const { data } = state;
+  const { backingData } = state;
   const { item } = action;
 
-  return { ...state, data: [item, ...data], modalKey: '', sortBy: '', searchTerm: '' };
+  return {
+    ...state,
+    data: [item, ...backingData.slice(0, backingData.length - 1)],
```

Comment:
oops I forgot about this hehe.

When adding an item - it should show in the list. If you're currently filtering, it would be weird that it showed in the filtered list. It would be weird if it did not show in the list. I think it's nice to add it to the top of the list.

So: clear the current filtering selection - make `data` have the new item, with a new array of all items in `backingData` except for the last element, which was shifted to the start of the array.

Strange bit of logic - I'll should have commented it. Let me know if this behaviour is correct in your head too - if you decide it is, I'll comment it hehe

Reply:
Nah I like. Do comment though

Reply:
1f812359cdb14877e869bbce78195f35ee54e65a